theano/gof/op.py

```diff
@@ -966,7 +966,7 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling):
         if new_node is not None:
             node = new_node
 
-        if self._op_use_c_code:
```

Comment:
OK, so the Op in question is `RandomFunction`, and indeed its `__getstate__` and `__setstate__` methods ignore `_op_use_c_code` (https://github.com/Theano/Theano/blob/master/theano/tensor/raw_random.py#L135-L149). I think we should fix it there.